chapters/ch48.md

```diff
@@ -1,55 +1,57 @@
 > # 48. More end-to-end learning examples
 
---> _replace THIS LINE by your translation for the above line_
+# 48. Thêm các ví dụ về phương pháp hoc bắt đầu - kết thúc
```

Comment:
- 'học đầu - cuối', cụm 'end-to-end learning' được dịch từ chương trước [47](https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/blob/master/chapters/ch47.md)
> phương pháp hoc bắt đầu - kết thúc

Reply:
@1612628 lưu ý không có dấu cách quay '-' của từ 'đầu-cuối'. Xem thêm https://github.com/aivivn/Machine-Learning-Yearning-Vietnamese-Translation/issues/334#issuecomment-556885151